src/makeRobot.js

```diff
@@ -38,7 +38,94 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    'name': name,
+    'chip version': version,
+    'wheels': wheels,
```

Comment:
```suggestion
    'name': name,
    'chip version': version,
    'wheels': wheels,
```
1 rule in checklist
https://github.com/mate-academy/js_task_make_robot/blob/master/checklist.md

Reply:
also don't make keys of object strings explicitly, there a strings by default